[feature] implement clean:s3 target

trombik opened the pull request:
This PR implements `clean:s3` target that removes all `OVA` files on S3.

also:

* document the inventory file

Commit:
implement `clean:s3` target to remove files on S3

Commit:
document the inventory file

fixes #18

Commit:
document `clean:s3`

Commit:
style

Commit:
update ToC

mitsururike commented on README.md:
```diff
@@ -149,11 +152,28 @@ upload:
+| `ostypeid` | value listed in `listOsTypes` API response |
+| `box_version` | version of the box |
+
+See `templates/inventories/default.yml`.
```
a link is preferable for users to see the file easily.

Commit:
add a href

https://github.com/reallyenglish/box2cloudstack/pull/19#pullrequestreview-72135503

Commit:
update template_name

also:

* replace service-dependant zone_name

probably other values are also service-dependant, but they clarify the
example.
